client/src/components/molecules/EditForm/ItemMiniEditForm.js

```diff
@@ -24,6 +28,59 @@ export const ItemMiniEditForm = ({
   const [uploadedImages, setUploadedImages] = useState(
     photos.sort((a, b) => b.front - a.front)
   );
+  const [selectedCategory, setSelectedCategory] = useState('');
+  const [selectedSubCategory, setSelectedSubCategory] = useState('');
+  const [item, setItem] = useState(null);
+
+  const formikProps = useFormikContext();
+
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
You are failing the lint due to this eslint-disable declaration being on the wrong line - it should be just above the dependency array on line 73... _But_ I think that also you can avoid having to use an eslint ignore declaration if you destructure `setFieldValue` directly from the formik hook like `const { setFieldValue } = useFormikContext();` and then include this in the dependency array like `[recordId, setFieldValue]`... this did not cause the cascading rerenders when I tried it...